src/main/kotlin/blackjack/model/BlackJack.kt

```diff
@@ -0,0 +1,19 @@
+package blackjack.model
+
+class BlackJack(val players: Players, cardDeck: CardDeck) {
+    private val cardDeck = cardDeck
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
class BlackJack(val players: Players, private val cardDeck: CardDeck) {
```